src/style-manager.js

```diff
@@ -22,19 +33,16 @@ export default class StyleManager {
     }
 
     generate() {
-        const _rules = {};
-
-        Object.keys(this.rules)
-            .map(key => {
-                const namespace = this.rules[key];
-                _rules[key] = {};
+        let _rules = Map();
```

Comment:
`rules` might be a misleading name here, since we are generating an Object/Map of namespaced styles.